parent/pom.xml

```diff
@@ -517,7 +517,7 @@
             <dependency>
                 <groupId>junit</groupId>
                 <artifactId>junit</artifactId>
-                <version>4.12</version>
+                <version>4.13.1</version>
```

Comment:
You need to update code to be able to do this (since e.g. `assertThat` and `ExpectedException` are deprecated in 4.13.1 and give warnings, which we fail on). I am working on this in another branch. If upgrading works while keeping 4.12 I would go with that.

Reply:
Ok, thanks. So ... the 4.13 that was bumped elsewhere probably broke this somewhere?